xclim/indices/stats.py

```diff
@@ -153,6 +155,62 @@ def fit(
     return out
 
 
+def fit_group(
+    da: xr.DataArray,
+    dist: str = "gamma",
+    freq: str = "MS",
+    window: int = 1,
+) -> xr.DataArray:
+    """Fit a resampled array averaged over a rolling window to a univariate distribution along the time dimension.
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+      Input data.
+    dist : str
+      Name of the univariate distribution, such as `beta`, `expon`, `genextreme`, `gamma`, `gumbel_r`, `lognorm`, `norm`
+      (see scipy.stats).
+    freq : str
+      Resampling frequency.
```

Comment:
```suggestion
    freq : str
      Resampling frequency. This function only makes sense for monthly or seasonal frequencies.
```

Reply:
I think my comment here relates to the one below about the resampling. But mainly, if we hardcode `time.month`, then frequencies other than `M' or 'Q' won't do what your are describing, no?  But, I did not think about the year+rolling case you mention, that would make sense here.

En fait, this function is only missing the capacity to fit over the `time.dayofyear` group. (And `time.week` but who does that anyway.)